Gemfile.lock

```diff
@@ -329,9 +329,6 @@ DEPENDENCIES
   jbuilder (~> 2)
   m (~> 1)
   minitest (~> 5.18)
-  net-imap
-  net-pop
-  net-smtp
```

Comment:
are't these required for ruby 3.2 +

Reply:
Maybe, but I think that can be fixed with a `bundle` call? Either way we'll see churn when running with different Ruby versions so I'll keep this in for now.